applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -533,14 +531,36 @@ static void fix_rep_wk(struct k_work *work)
 		.tm_min  = pvt.datetime.minute,
 		.tm_sec  = pvt.datetime.seconds,
 	};
+#endif
+
+#if defined(CONFIG_SLM_NRF_CLOUD)
+
+	if (slm_nrf_cloud_send_location && slm_nrf_cloud_ready) {
+		/* Report to nRF Cloud by best-effort */
+		send_location(&pvt);
+	}
 
+#if defined(CONFIG_NRF_CLOUD_PGPS) && defined(CONFIG_SLM_PGPS_INJECT_FIX_DATA)
```

Comment:
As you removed it from the previous `#ifdef`, might as well here:
```suggestion
#if defined(CONFIG_SLM_PGPS_INJECT_FIX_DATA)
```